cppwinrt/cmd_reader.h

```diff
@@ -195,11 +195,10 @@ namespace cppwinrt
     inline std::string get_module_path()
     {
         std::string path(100, '?');
-        DWORD actual_size{};
 
-        while (true)
+        for (;;)
```

Comment:
I don't see how `for (;;)` is any clearer/better than `while (true)`

Reply:
Yes, I prefer `while (true)` but I assumed Clang warns about this. Is that correct?

Reply:
Doesn't seem so: https://godbolt.org/z/GE9cW8v3z

Reply:
Sweet, then I'd prefer to keep the while loops.